core/txpool/legacypool/list.go

```diff
@@ -344,6 +344,10 @@ func (l *list) Add(tx *types.Transaction, priceBump uint64, l1CostFn txpool.L1Co
 	l.totalcost.Add(l.totalcost, cost)
 	if l1CostFn != nil {
 		if l1Cost := l1CostFn(tx.RollupCostData()); l1Cost != nil { // add rollup cost
+			cost, overflow := uint256.FromBig(l1Cost)
```

Comment:
Should this be modifying line 351 to be `l.totalcost.Add(l.totalcost, l1Cost)` ?